common/oauthTokenManager.go

```diff
@@ -434,6 +448,17 @@ type OAuthTokenInfo struct {
 	ClientID string `json:"_client_id"`
 }
 
+func (t *OAuthTokenInfo) Expires() time.Time {
+	s, err := t.ExpiresOn.Float64()
+	if err != nil {
```

Comment:
why are we overwriting the expiry time if there is no error?

Reply:
We relocated this function to a new location since its previous file was removed due to non-usage.